src/common/mailing.ts

```diff
@@ -173,7 +175,10 @@ export const updateUserContactLists = async (
       lists.push(LIST_MARKETING_EMAILS);
     } else if (contactId) {
       // If they no longer subscribe to marketing emails remove them from the contact list
-      await removeUserFromList(LIST_MARKETING_EMAILS, contactId);
+      await Promise.all([
+        removeUserFromList(LIST_MARKETING_EMAILS, contactId),
+        removeUserFromList(LIST_DRIP_CAMPAIGN, contactId),
```

Comment:
Remove when user did not accept marketing campaign [per this comment](https://dailydotdev.slack.com/archives/C03QZBF46JC/p1693838974151479?thread_ts=1692689849.334919&cid=C03QZBF46JC)